packages/vsce/src/doc/meta/JVMServer.meta.ts

```diff
@@ -0,0 +1,63 @@
+import { CicsCmciConstants } from "@zowe/cics-for-zowe-sdk";
+import { Resource } from "../../resources/Resource";
+import { PersistentStorage } from "../../utils/PersistentStorage";
+import { IResourceMeta } from "./IResourceMeta";
+import { IJVMServer } from "../resources/IJVMServer";
+
+const persistentStorage = new PersistentStorage("zowe.cics.persistent");
+
+export const JVMServerMeta: IResourceMeta<IJVMServer> = {
+  resourceName: CicsCmciConstants.CICS_JVMSERVER_RESOURCE,
+  humanReadableName: "JVM Servers",
+
+  buildCriteria(criteria: string[]) {
+    return criteria.map((n) => `NAME=${n}`).join(" OR ");
+  },
+
+  async getDefaultCriteria() {
+    return PersistentStorage.getDefaultFilter(CicsCmciConstants.CICS_JVMSERVER_RESOURCE, "jvmServer");
+  },
+
+  getLabel: function (jvmServer: Resource<IJVMServer>): string {
+    let label = `${jvmServer.attributes.name}`;
+    if (jvmServer.attributes.enableStatus && jvmServer.attributes.enableStatus.toLowerCase() === "disabled") {
+      label += " (disabled)";
```

Comment:
To match all other metas.
```suggestion
      label += " (Disabled)";
```